### DISCUSSION TITLE
自动播放分集bug

### DISCUSSION THREAD

**Discussion by JiangSNZero** [xfangfang/wiliwili#128] [Ideas]
使用环境为Windows10 家庭版 21H2，在应用程序窗口最小化的情况下自动播放分集的功能会失效。

**Comment by xfangfang** [comment]
@CopyMasterJ 这个很难说是bug还是feature。
问题出现的原因是，最小化之后主循环会被卡住，导致接收不到播放器传出的各类事件。
这个问题我想大概率保持现状了，目前的重点还是完善功能 (而且这个现象貌似也还挺合理的哈)

**Reply by xfangfang** [reply]
这个看起来并不是很容易解决的问题，我暂时也不想花大量的时间重构代码来解决这个问题。（“性价比”不高）
不过还是欢迎各路大神提交PR
